src/ngx_http_lua_module.c

```diff
@@ -661,6 +679,64 @@ ngx_http_lua_init(ngx_conf_t *cf)
         lmcf->postponed_to_access_phase_end = 0;
     }
 
+    len = lmcf->lua_cpath.len;
+
+    if (lmcf->requires_append_lua_path) {
+        for (i = 0; i < lmcf->append_lua_cpath->nelts; i++) {
+            p = (ngx_str_t *) lmcf->append_lua_cpath->elts;
+            len += p[i].len;
+        }
+
+        path = (char *) ngx_palloc(cf->pool, len);
```

Comment:
Since you'll call `ngx_memzero()` on the memory block later anyway, why not just use `ngx_pcalloc` here instead?